tests/fixtures.py

```diff
@@ -141,3 +142,14 @@ def init_data():
             start_at=date.today(), end_at=one_month_from_now, amount=3600)
         db.session.add(zero_discount_price)
         db.session.commit()
+
+        signed = DiscountPolicy(title='Signed',
+            item_quantity_min=1,
+            discount_type=DISCOUNT_TYPE.COUPON,
+            percentage=10,
+            discount_code_base='signed',
+            secret=uuid.uuid4().hex,
+            organization=rootconf)
```

Comment:
This should perhaps be a classmethod that constructs signed discount policies, like `DiscountPolicy.new_bulk()` so that `secret` isn't left to the caller. Also, "bulk" is a more user friendly term than "signed".